dataland-backend/src/main/kotlin/org/dataland/datalandbackend/api/DataOwnerApi.kt

```diff
@@ -0,0 +1,129 @@
+package org.dataland.datalandbackend.api
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import io.swagger.v3.oas.annotations.security.SecurityRequirement
+import org.dataland.datalandbackend.model.companies.CompanyDataOwners
```

Comment:
![image](https://github.com/d-fine/Dataland/assets/104447806/67cde723-8d8c-4f05-b647-13239e74bf01)
If i get the owners of a company without owners the api throws an error. If i compare it to the data get endpoints if i try to get a dataset of framework, for which the company has no dataset, i receive a 200 error with an empty responsebody. The same behaviour i would have expected here, if a company simply has no data owner.

Reply:
changed now

Reply:
ok